### PR TITLE
Apache MXNet rename

### PR DESCRIPTION
**Pull request by msmykx-intel**
MXNet renaming into Apache MXNet

### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md [hunk 1]
```diff
@@ -624,10 +624,10 @@ If a `*.caffemodel` file exists and is correct, the error occurred possibly beca
 
 #### 100. What does the message "SyntaxError: 'yield' inside list comprehension" during MxNet model conversion mean? <a name="question-100"></a>
 
-The issue "SyntaxError: `yield` inside list comprehension" might occur during converting MXNet models (mobilefacedet-v1-mxnet, brain-tumor-segmentation-0001) on Windows platform with Python 3.8 environment. This issue is caused by the API changes for `yield expression` in Python 3.8.
+The issue "SyntaxError: `yield` inside list comprehension" might occur during converting MXNet models (`mobilefacedet-v1-mxnet`, `brain-tumor-segmentation-0001`) on Windows platform with Python 3.8 environment. This issue is caused by the API changes for `yield expression` in Python 3.8.
 The following workarounds are suggested to resolve this issue:
 1. Use Python 3.6/3.7 to convert MXNet models on Windows
-2. Update MXNet by using `pip install mxnet=1.7.0.post2`
+2. Update Apache MXNet by using `pip install mxnet=1.7.0.post2`
```
Comment (helena-intel) [inline docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md:630]:
```suggestion
2. Update Apache MXNet by using `pip install mxnet==1.7.0.post2`
```
Error in original text

**Review by helena-intel — APPROVED** [review]
Thank you, also for removing the asterisks! MXNet naming should probably be updated in OMZ READMEs too. @kblaszczak-intel @xu-yuan1

File: docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md (hunk 1 above)
Comment (msmykx-intel) [reply docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md]: No problem. I'm still working on renaming in other files/articles, so I'll regularly update this PR :)

**Review by kblaszczak-intel — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md [hunk 2]
```diff
@@ -22,17 +22,17 @@ MXNet-specific parameters:
...
             Enable transformation for converting the gluoncv ssd topologies.
             Use only if your topology is one of ssd gluoncv topologies
 ```
 
-> **NOTE**: By default, Model Optimizer does not use the MXNet loader. It transforms the topology to another format which is compatible with the latest
-> version of MXNet. However, the MXNet loader is required for models trained with lower version of MXNet. If your model was trained with an MXNet version lower than 1.0.0, specify the
-> `--legacy_mxnet_model` key to enable the MXNet loader. Note that the loader does not support models with custom layers. In this case, you must manually
-> recompile MXNet with custom layers and install it in your environment.
+> **NOTE**: By default, Model Optimizer does not use the Apache MXNet loader. It transforms the topology to another format which is compatible with the latest
+> version of Apache MXNet. However, the Apache MXNet loader is required for models trained with lower version of Apache MXNet. If your model was trained with an Apache MXNet version lower than 1.0.0, specify the
```
Comment (anko-intel) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md:33]: Only the first occurrence on the page must be the official name "Apache MXNet". Subsequent ones can be left as MXNet, but using always "Apache MXNet" is also fine.

Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md:33]: It will be easier to keep track of the corrected pages this way ;)

Comment (helena-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md:33]: Having this longer name five times in a short sentence does impact readability though :-( (maybe more so if you are familiar with MXNet). I'm fine with leaving it this way, but in general, surely we can search for pages that do include MXNet and do not include "Apache MXNet"? Also I assume this is not a problem, but for these releases lower than 1.0.0 we mention, the product was not called Apache MXNet but just MXNet.

Comment (anko-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md:33]: Even if MXNet 1.0 was called itself MXNet - it was wrong according to Apache. It should had been called "Apache MXNet (Incubating)"

**Pull request merged**